### PR TITLE
improving eventing doc (?)

### PR DESCRIPTION
**Pull request by lionelvillard**
## Proposed Changes

- Consistent use of API Server Source instead of Kubernetes Events Source
- Add warning about Minikube

### CODE REVIEW

**Comment by lionelvillard** [comment]
@vaikas

File: docs/eventing/README.md [hunk 1]
```diff
@@ -156,22 +156,11 @@ If your code needs to send events as part of its business logic and doesn't fit
 the model of a Source, consider
 [feeding events directly to a Broker](https://knative.dev/docs/eventing/broker-trigger/#manual).
 
-### KubernetesEventSource
+### APIServerSource
```
Comment (matzew) [inline docs/eventing/README.md]: :+1:

File: docs/eventing/getting-started.md [hunk 2]
```diff
@@ -19,17 +19,20 @@ Use this guide to learn how to create, send, and verify events in Knative. The s
...
 - [`kubectl` CLI tool](https://kubernetes.io/docs/reference/kubectl/overview/) within a minor version of your Kubernetes cluster.
 
-- [curl v7.65 or higher](https://curl.haxx.se/download.html)
+- [curl v7.65 or higher](https://curl.haxx.se/download.html).
 
-- Knative Eventing Component
+- Knative Eventing component.
 
-   - Knative Eventing In-memory channel 
+- Knative Eventing in-memory channel.
```
Comment (samodell) [inline docs/eventing/getting-started.md]: Hm, why shouldn't the in-memory channel be nested under the Eventing component? I thought a pre-req for the IMC was having Knative Eventing installed.

Comment (lionelvillard) [reply docs/eventing/getting-started.md]: fixed, thx :-)

**Pull request merged**